docs/configuration.md

```diff
@@ -181,10 +181,15 @@ of the most common options to set are:
   <td><code>spark.driver.memoryOverhead</code></td>
   <td>driverMemory * 0.10, with minimum of 384 </td>
   <td>
-    The amount of off-heap memory to be allocated per driver in cluster mode, in MiB unless
-    otherwise specified. This is memory that accounts for things like VM overheads, interned strings, 
-    other native overheads, etc. This tends to grow with the container size (typically 6-10%). 
-    This option is currently supported on YARN, Mesos and Kubernetes.
+    Amount of non-heap memory to be allocated per driver process in cluster mode 
+    (e.g YARN, Mesos and Kubernetes.), in MiB unless otherwise specified. This is memory that
+    accounts for things like VM overheads, interned strings, other native overheads, etc. 
+    This tends to grow with the container size (typically 6-10%). 
+    <em>Note:</em> Non-heap memory including off-heap memory 
```

Comment:
Hm, what else gets counted towards the driver's memory here, in a YARN context? are you thinking of things like the Python process that goes with a Pyspark driver? I don't think this is wrong but at best that's the additional comment to make here.

Reply:
Thanks for your reminder. I have extended comment about PySpark.